Makefile

```diff
@@ -10,6 +10,9 @@ test: node_modules
 node_modules:
 	npm install
 
+build:
+	npm run build
+
 publish: test version upload unversion
```

Comment:
@camiblanch @maxwellpothier  I agree with reducing the package size. When I first looked at the project, I wasn't sure why all the code was being exported in the package, so I left it as it was. However, in this case, it would be better to specify which files to include rather than excluding them. In the package.json, you can do something like this:

```
{
  "files": [
    "dist",
    "README.md",
    "LICENSE"
  ]
}
```

This way, you’re only including the necessary build folder (dist), which is meant for publishing the package to npm, and other relevant files like the README and license.